src/shared/select-utilities.ts

```diff
@@ -325,8 +325,15 @@ export const selectFavoriteFilters = async (): Promise<IFavouriteFilter | undefi
             favFilters
               .filter(filter => {
                 let valid = true;
-                otherProjects.forEach(prj => {
-                  if (filter.jql.indexOf(prj.key) >= 0 || filter.jql.indexOf(prj.name) >= 0) {
+                otherProjects.forEach(prj => 
+                {
+                  let regexp0 = new RegExp('project = "?' + prj.key + '"?');
```

Comment:
Really nice catch. 👍This is the right approach.
In my Jira I can also use `!=, is, is not, not in` for project field, do you think we must manage also them?
We need also to use case insensitive RegExp.